### DISCUSSION TITLE
i am unable to add my code to git hub repository.Please go through the screen shot

### DISCUSSION THREAD

**Discussion by AnilkrishnaD** [community/community#35749] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/113106662/195056866-649c4e5c-10ac-4df0-a0a9-7fe1677edf9f.png)

**Comment by AF250329** [comment]
I am not sure but I see a line that says: `fatal: Authentication failed for http://github.com/.......` - does it mean anything to you ? May be you can check your password/username ? Double check it ?

**Comment by MitkoVtori** [comment]
I don't understand what is the problem but here's an easier way to commit & push your changes without commands:
![Screenshot 2022-10-11 143537](https://user-images.githubusercontent.com/112943652/195080227-b248d65c-752a-4047-a359-cf0eb68ad4ef.png)
![Screenshot 2022-10-11 143817](https://user-images.githubusercontent.com/112943652/195080253-ec0b49ae-4b2f-4300-8028-795e82dbf708.png)

**Reply by MitkoVtori** [reply]
If there's something about this way of pushing you can ask at anytime.

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
